relationship_server.py

```diff
@@ -132,9 +132,18 @@ def process_records(records):
 
 @app.route("/", methods=["POST"])
 def single():
-    req_data = request.get_json()
+    try:
+        req_data = request.get_json()
+        results = process_records(req_data)
+    except TypeError:
+        # abort when not JSON
+        abort(400)
```

Comment:
There are import errors with abort and jsonify. 
Please import `from flask import jsonify, abort`

Reply:
I'm sorry there were uncommitted changes in my repo that I forgot about!